src/coreclr/src/jit/target.h

```diff
@@ -6,7 +6,7 @@
 #ifndef TARGET_H_
 #define TARGET_H_
 
-#if defined(TARGET_UNIX)
+#if defined(TARGET_UNIX) || defined(TARGET_ARM)
```

Comment:
Flip the logic to: `#if defined(TARGET_WINDOWS) && !defined(TARGET_ARM)` to make it look like the condition in other places.